fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/utils/iterators/PersistingRdfStreamConsumer.java

```diff
@@ -89,13 +89,15 @@ public boolean apply(final Triple t) {
                             "Discovered a Fedora-relevant subject in triple: {}.",
                             t);
                 } else {
-                    LOGGER.debug("Ignoring triple: {}.", t);
+                    LOGGER.error("subject ({}) is not in repository domain.", t.getSubject().toString());
+                    throw new MalformedRdfException(String.format(
+                        "RDF Stream contains subject(s) (%s) not in the domain of this repository.", t.getSubject()));
```

Comment:
I think we see stuff like this elsewhere. In another PR, it might be worth adding a constructor `RepositoryRuntimeException(String format, Object... templateValues)`.